InnoGotchi.API/Tests/Repositories/UserRepositoryTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Infrastructure.Persistance;
+using Infrastructure.Persistance.Repositories;
+using InnoGotchi.Domain.Entities;
+
+namespace Tests.Repositories
+{
+    public class UserRepositoryTests : IClassFixture<TestDatabaseFixture>
+    {
+        private readonly Fixture _fixture;
+        private readonly AppDbContext _context;
+        private readonly UserRepository _repository;
+        public UserRepositoryTests(TestDatabaseFixture fixture)
+        {
+            _fixture = new Fixture();
+            _context = fixture.Context;
+            _repository = new UserRepository(_context);
+        }
+
+        [Fact]
+        public async Task GetUsersAsync_ReturnsUsersList()
+        {
+            //Arrange
+
+            //Act
+            var result = await _repository.GetUsersAsync(false);
+
+            //Assert
+            result.Should().NotBeNull()
+                .And.HaveCount(3);
+        }
+
+        [Fact]
+        public async Task GetUserByIdAsync_ReturnsUser_WhenUserExistsAndPassedIdIsValid()
```

Comment:
Usually the 2nd and the 3rd parts are changed: GetUserByIdAsync_WhenUserExistsAndPassedIdIsValid_ReturnsUser

Reply:
renamed